### ISSUE TITLE
Running tests does not use REFERENCE_SCHEMA in tenant creation

### ISSUE THREAD

**Issue by ZipBrandon** [lorinkoz/django-pgschemas#135]
I'm finally getting around to putting some Django tests. I am successfully using a `REFERENCE_SCHEMA` in my project but I am not seeing it clone when it running with `manage.py test --keepdb`.  My expectation is that the `test` schema/tenant would be using my clone schema. It runs the regular migrations route instead.  I visited the docs but I couldn't find anything to help resolve this. Could this be a misconfiguration on my part?

**Comment by ZipBrandon** [comment]
Well I discovered the issue https://github.com/lorinkoz/django-pgschemas/blob/83ce13044c20c2a390b6874dc875462688153cfd/django_pgschemas/utils.py#L213 but am unsure why this guard is present for test mode.

**Comment by lorinkoz** [comment]
Sorry for the super late reply. Yes, that's pretty much expected as of now, and if your main concern is how slow tests currently are, you are totally right. One of the areas I've struggled the most with is coming up with proper setup for testing. I could use your input in how to shape the tenant test cases in the future. Here are some options.

- Tenants get set up up via test class hooks (current approach `TestCase.setup_tenant`, `TestCase.`) migrated from zero or cloned from the reference schema.
- A single tenant is set up via centralized function (e.g. `TenantModel.setup_tenant_for_test`) and reused across all tests of the same kind (should be super fast), migrated from zero.

I don't remember if I found any issues with cloning the reference schema for testing. I could try to remove the limitation in the code and see what happens. Won't have much time in the coming weeks unfortunately so if you want to do some research yourself, happy to follow up on that.
